lib/src/chat_view.dart

```diff
@@ -420,120 +420,127 @@ class DashChatState extends State<DashChat> {
-            user: widget.user,
-            messages: widget.messages,
-            showuserAvatar: widget.showUserAvatar,
-            dateFormat: widget.dateFormat,
-            timeFormat: widget.timeFormat,
-            inverted: widget.inverted,
-            showAvatarForEverMessage: widget.showAvatarForEveryMessage,
-            onLongPressAvatar: widget.onLongPressAvatar,
-            onPressAvatar: widget.onPressAvatar,
-            onLongPressMessage: widget.onLongPressMessage,
-            avatarBuilder: widget.avatarBuilder,
-            messageBuilder: widget.messageBuilder,
-            messageTextBuilder: widget.messageTextBuilder,
-            messageImageBuilder: widget.messageImageBuilder,
-            messageTimeBuilder: widget.messageTimeBuilder,
-            dateBuilder: widget.dateBuilder,
-            messageContainerDecoration: widget.messageContainerDecoration,
-            parsePatterns: widget.parsePatterns,
-            changeVisible: changeVisible,
-            visible: visible,
-            showLoadMore: showLoadMore,
-          ),
-          if (widget.messages.length != 0 &&
-              widget.messages[widget.messages.length - 1].user.uid !=
-                  widget.user.uid)
-            Container(
-              constraints: BoxConstraints(maxHeight: 100.0),
-              width: MediaQuery.of(context).size.width,
-              child: Wrap(
-                children: <Widget>[
-                  if (widget
-                          .messages[widget.messages.length - 1].quickReplies !=
-                      null)
-                    ...widget.messages[widget.messages.length - 1].quickReplies
-                        .values
-                        .sublist(
-                            0,
-                            widget.messages[widget.messages.length - 1]
-                                        .quickReplies.values.length <=
-                                    3
-                                ? widget.messages[widget.messages.length - 1]
-                                    .quickReplies.values.length
-                                : 3)
-                        .map(
-                          (reply) => QuickReply(
-                            reply: reply,
-                            onReply: widget.onQuickReply,
-                            quickReplyBuilder: widget.quickReplyBuilder,
-                            quickReplyStyle: widget.quickReplyStyle,
-                            quickReplyTextStyle: widget.quickReplyTextStyle,
-                          ),
-                        )
-                        .toList(),
-                ],
-              ),
+    return LayoutBuilder(
+      builder: (context, constraints) {
+        final maxWidth = constraints.maxWidth == double.infinity ? MediaQuery.of(context).size.width : constraints.maxWidth;
+        final maxHeight = constraints.maxWidth == double.infinity ? MediaQuery.of(context).size.height : constraints.maxWidth;
```

Comment:
This should be `maxHeight`

Reply:
Other than this everything looks good

Reply:
Cool :) good catch ! I just push the fix :)